code/datums/gamemode/role/cultist.dm

```diff
@@ -208,86 +208,86 @@
 	return dat
 
 /datum/role/cultist/handle_reagent(var/reagent_id)
-	switch (reagent_id)
-		if (HOLYWATER)
-			var/mob/living/carbon/human/H = antag.current
-			if (!istype(H))
-				return
-			var/unholy = H.checkTattoo(TATTOO_HOLY)
-			var/current_act = max(-1,min(5,veil_thickness))
-			if (holywarning_cooldown <= 0)
-				holywarning_cooldown = 5
-				if (unholy)
-					to_chat(H, "<span class='warning'>You feel the unpleasant touch of holy water, but the mark on your back negates its most debilitating effects.</span>")
-				else
-					switch (current_act)
-						if (CULT_MENDED)
-							to_chat(H, "<span class='danger'>The holy water permeates your skin and consumes your cursed blood like mercury digests gold.</span>")
-						if (CULT_PROLOGUE)
-							to_chat(H, "<span class='warning'>You feel the cold touch of holy water, but the veil is still too thick for it to be a real threat.</span>")
-						if (CULT_ACT_I)
-							to_chat(H, "<span class='warning'>The touch of holy water troubles your thoughts, you won't be able to cast spells under its effects.</span>")
-						if (CULT_ACT_II)
-							to_chat(H, "<span class='danger'>The holy water makes your head spin, you're having trouble walking straight.</span>")
-						if (CULT_ACT_III)
-							to_chat(H, "<span class='danger'>The holy water freezes your muscles, you find yourself short of breath.</span>")
-						if (CULT_ACT_IV)
-							to_chat(H, "<span class='danger'>The holy water makes you sick to your stomach.</span>")
-						if (CULT_EPILOGUE)
-							to_chat(H, "<span class='danger'>Even in these times, holy water proves itself capable of hindering your progression.</span>")
-
+	var/mob/living/carbon/human/H = antag.current
+	if (!istype(H))
+		return
+	var/unholy = H.checkTattoo(TATTOO_HOLY)
+	var/current_act = max(-1,min(5,veil_thickness))
+	if (reagent_id == HOLYWATER)
```

Comment:
for this case in particular, it's because of the second check at L.238 and I'd rather not have to copypaste the code for both Holywater and Harebell incense, but I don't want the flavor text to show up for incense.